core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -85,8 +85,8 @@ final case class EitherT[F[_], A, B](value: F[Either[A, B]]) {
                                                          applicativeG: Applicative[G]): G[EitherT[F, C, D]] =
     applicativeG.map(traverseF.traverse(value)(axb => Bitraverse[Either].bitraverse(axb)(f, g)))(EitherT.apply)
 
-  def biflatMap[AA >: A, BB >: B](fa: A => EitherT[F, AA, BB],
-                                  fb: B => EitherT[F, AA, BB])(implicit F: FlatMap[F]): EitherT[F, AA, BB] =
+  def biflatMap[AA, BB](fa: A => EitherT[F, AA, BB],
```

Comment:
Thanks! Good catch. Would you change these type param names to `C` and `D` to be more consistent? 
Actually there is one more here
https://github.com/typelevel/cats/blob/5ca98e71a44cc0e0bbc060392c677a64f545a7b8/core/src/main/scala/cats/data/EitherT.scala#L34, change that `AA`to `C` will make it more consistent as well.